### PR TITLE
fix(server): standardizing datastore resource names

### PR DESCRIPTION
**Pull request by danielbdias**
This PR updates the names of the datastore resource files to a common standard that we defined with Tracetest Core contributors.

## Changes

- Updating datastore resource file names and references

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: server/openapi/api_resource_api_service.go [hunk 1]
```diff
@@ -192,8 +192,8 @@ func (s *ResourceApiApiService) GetDataStore(ctx context.Context, dataStoreId st
 	// TODO - update GetDataStore with the required logic for this service method.
 	// Add api_resource_api_service.go to the .openapi-generator-ignore to avoid overwriting this service implementation when updating open api generation.
 
-	//TODO: Uncomment the next line to return response Response(200, DataStoreResource{}) or use other options such as http.Ok ...
-	//return Response(200, DataStoreResource{}), nil
+	//TODO: Uncomment the next line to return response Response(200, datastore{}) or use other options such as http.Ok ...
```
Comment (schoren) [inline server/openapi/api_resource_api_service.go:195]: should this be renamed? try running `make generate` to keep it in sync with the openapi files

File: server/openapi/model_data_store_list.go [hunk 2]
```diff
@@ -12,7 +12,7 @@ package openapi
 type DataStoreList struct {
 	Count int32 `json:"count,omitempty"`
 
-	Items []DataStoreResource `json:"items,omitempty"`
+	Items []datastore `json:"items,omitempty"`
```
Comment (schoren) [inline server/openapi/model_data_store_list.go:15]: same here

File: server/openapi/api_resource_api_service.go (hunk 1 above)
Comment (danielbdias) [reply server/openapi/api_resource_api_service.go]: No 🤦 , I'm reverting this change.

File: server/openapi/model_data_store_list.go (hunk 2 above)
Comment (danielbdias) [reply server/openapi/model_data_store_list.go]: Reverting too.

**Review by schoren — APPROVED** [review]

**Pull request merged**